panther_analysis_tool/util.py

```diff
@@ -173,3 +173,9 @@ def convert_unicode(obj: Any) -> str:
     e.g. \\\\u003c => <"""
     string_to_convert = str(obj)
     return re.sub(r"\\*\\u([0-9a-f]{4})", lambda m: chr(int(m.group(1), 16)), string_to_convert)
+
+
+def is_simple_detection(analysis_item: Dict[str, Any]) -> bool:
+    return (
+        analysis_item.get("Detection") is not None or analysis_item.get("InlineFilter") is not None
```

Comment:
hmm should we leave out the InlineFilter part until we have support for that? IIRC presence of inline filters does not guarantee that the detection is a simple detection

Reply:
Done [77084ba](https://github.com/panther-labs/panther_analysis_tool/pull/284/commits/77084ba3d16fcc3d2168939116ebe7951e536868)

Reply:
thanks!